docs/contribute/intro.md

```diff
@@ -44,19 +44,20 @@ code looks correct according to a few checks.
 
 ### Run the tests
 
-For information about running tests, see {ref}`developer/tests`.
+For code tests, jupyter-book uses [pytest](https://docs.pytest.org)).
+You may run the tests with the following command:
 
-(developer/tests)=
-## Testing infrastructure
+```shell
+>> pytest --ignore=tests/test_pdf.py
```

Comment:
@chrisjsewell are we still ignoring `test_pdf`? I thought you fixed that recently on `CI` or is this the suggested way to run the tests locally to avoid dependencies on `tex`?

Reply:
btw - I see the text below about LaTeX - but is that the recommended way to run tests locally you think?

Reply:
Yeh I think it works well enough, I guess the main thing is just to note that you need TeX installed to have certain tests pass.
But actually it would be better to decorate these tests with `@pytest.mark.requires_tex`, then you would run `pytest -k 'not requires_tex`

Reply:
I think I'll do this in a different PR, since its not a priority